getting-started-guides/on-prem-cluster/standalone-python.md

```diff
@@ -181,13 +162,14 @@ ${SPARK_HOME}/bin/spark-submit
  --driver-memory ${SPARK_DRIVER_MEMORY}                                         \
  --executor-memory ${SPARK_EXECUTOR_MEMORY}                                     \
  --conf spark.cores.max=${TOTAL_CORES}                                          \
- --jars ${SPARK_JARS},${JAR_RAPIDS}                                                           \
- --py-files ${SPARK_PY_FILES}                                                   \
+ --conf spark.executor.extraClassPath=${CUDF_JAR}:${RAPIDS_JAR}  \
+ --jars ${CUDF_JAR},${RAPIDS_JAR},${XGBOOST4J_JAR},${XGBOOST4J_SPARK_JAR}     \
```

Comment:
Same issue as above ones. Without the full path, how can this command find the jars?